File: Python/1_Bubble Sort/Bubble Sort.py
def bubble_sort(n_list):
    for i in range(len(n_list)-1, 0, -1):
        for j in range(i):
            if n_list[j] > n_list[j+1]:
                n_list[j], n_list[j+1] = n_list[j+1], n_list[j]

n_list = [14, 46, 43, 27, 57, 41, 45, 21, 70]
bubble_sort(n_list)
print(n_list)

File: Swift/1_Bubble Sort/Bubble Sort.swift
func bubbleSort(arr: [Int]) -> [Int]{
	var array = arr
	for _ in 0..<array.count - 1 {
		for j in 0..<array.count - 1 {
			if array[j] > array[j+1] {
				let temp = array[j]
				array[j] = array[j+1]
				array[j+1] = temp
			}
		}
	}
	return array
}

let unsortedArray = [14, 46, 43, 27, 57, 41, 45, 21, 70]
print(bubbleSort(arr: unsortedArray))